src/transformers/processing_utils.py

```diff
@@ -317,6 +317,7 @@ def get_processor_dict(
                     user_agent=user_agent,
                     revision=revision,
                     subfolder=subfolder,
+                    _raise_exceptions_for_missing_entries=False,
```

Comment:
I might be missing something, but I don't understand how this fixes things, as this argument doesn't seem to be used anywhere else in this file

Reply:
It's an argument to the function `cached_file`  defined in another file.

Using `_raise_exceptions_for_missing_entries=False` when calling `cached_file`, it won't fail if the target file is missing.

Let me know if you need more context/info about it.

Reply:
d'oh sorry, I misread as it being an argument into `def get_processor_dict(` because of the diff. My bad!